includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -28,6 +28,7 @@ class WC_Coupon_Data_Store_CPT extends WC_Data_Store_WP implements WC_Coupon_Dat
 		'discount_type',
 		'coupon_amount',
 		'expiry_date',
+		'expiry_date_utc',
```

Comment:
Gonna be a pain :) Since `expiry_date` is legacy, the new field can actually just use the name of the prop like we did in orders. So rather than `expiry_date_utc`, just name it `date_expires` to match the prop being loaded. That's more consistent IMO.